proposals/secure-storage-polyfill.js

```diff
@@ -0,0 +1,67 @@
+// Existing browser APIs don't give us access to the system's secure storage,
+// meaning all data in this polyfill is stored in less secure mechanisms.
+console.warn(
+  "Warning: browser.secureStorage polyfill loaded. This proof of concept stores data insecurely and should not be used in production."
+);
+
+const RECOGNISED_AUTH_METHODS = [
+  "PIN",
+  "PASSWORD",
+  "BIOMETRY_FACE",
+  "BIOMETRY_FINGERPRINT",
+];
+
+window.secureStorage = {
```

Comment:
Should this go on the `browser` and `chrome` objects to be an accurate polyfill?

Reply:
Good call! I assumed without checking that these would be read-only, but I was totally wrong. I've updated this to attach to both of those where available.